rocAL/rocAL/source/loaders/video/video_loader.cpp

```diff
@@ -243,26 +244,26 @@ VideoLoader::update_output_image()
     {
         auto data_buffer = _circ_buff.get_read_buffer_dev();
         _swap_handle_time.start();
-        if (_output_image->swap_handle(data_buffer) != 0)
+        if (_output_tensor->swap_handle(data_buffer) != 0)
             return LoaderModuleStatus ::DEVICE_BUFFER_SWAP_FAILED;
         _swap_handle_time.end();
     }
     else
     {
         auto data_buffer = _circ_buff.get_read_buffer_host();
         _swap_handle_time.start();
-        if (_output_image->swap_handle(data_buffer) != 0)
+        if (_output_tensor->swap_handle(data_buffer) != 0)
             return LoaderModuleStatus::HOST_BUFFER_SWAP_FAILED;
         _swap_handle_time.end();
     }
     if (_stopped)
         return LoaderModuleStatus::OK;
     _output_decoded_img_info = _circ_buff.get_image_info();
     _output_names = _output_decoded_img_info._image_names;
-    _output_image->update_image_roi(_output_decoded_img_info._roi_width, _output_decoded_img_info._roi_height);
+    _output_tensor->update_tensor_roi(_output_decoded_img_info._roi_width, _output_decoded_img_info._roi_height);
     _circ_buff.pop();
     if (!_loop)
-        _remaining_sequences_count -= _sequence_count;
+        _remaining_sequences_count -= _batch_size;
```

Comment:
is the sequence_count and batch_size the same

Reply:
In the current tensor pipeline we have made the Video pipeline similar to image pipeline to operate on the batch size.
Here the sequence_count is the same as batch size in tensor pipeline.